src/assets/templates/widgets/profile/bconnected_service_loop.html

```diff
@@ -17,23 +33,35 @@
         data-ng-click="showValidation = true"
         data-ng-disabled="api.user.profile['actingAsUid'] || api.user.profile['delegateActingAsUid'] || api.user.profile['advisorActingAsUid']"
         data-ng-show="!showValidation"
-        class="cc-button">
+        class="cc-button"
+      >
         Disconnect <span class="cc-visuallyhidden">Google access</span>
       </button>
     </div>
     <div data-ng-show="showValidation">
-      <strong class="cc-profile-bconnected-service-sure-question">Are you sure?</strong>
+      <strong class="cc-profile-bconnected-service-sure-question"
+        >Are you sure?</strong
```

Comment:
@redconfetti this formating looks funky

Reply:
I'm pretty sure this is just an awkward result of our prettier configuration. HTML is formatted a little more "React" style, which can lead to some of those weird `>`s at the start of lines